hydro/distribution.yaml

```diff
@@ -1875,6 +1875,11 @@ repositories:
       type: git
       url: https://github.com/kth-ros-pkg/force_torque_tools.git
       version: hydro
+  foscam_8918_driver:
+    doc:
+      type: git
+      url: https://github.com/tdenewiler/foscam_8918_driver
+      version: hydro-devel    
```

Comment:
The CI is failing because this has trailing whitespace, can you please remove it?